src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from '@patternfly/react-core';
+import { Icon } from 'patternfly-react';
+
+import { getHostStatusText } from '../../selectors';
+
+export const IconAndText = ({ icon, text }) => (
+  <span className="co-icon-and-text">
```

Comment:
can you create a new style which conforms to BEM ?

Reply:
I've re-used the `.co-icon-and-text__icon` class to add the margin on the right.

Reply:
I have checked and using wrapping span with `co-icon-and-text` class has no impact on styling so it can be removed completely.

For the icon itself, you can keep the className which adds the margin or remove the className and just add space between Icon and text which does the same.

`<Icon type="pf" name={icon} /> {text}`

Reply:
The class creates a margin that is a tiny bit wider than a space, so I kept it to keep it consistent with other tables in the console.